README.md

```diff
@@ -23,4 +25,17 @@ Run with `go test -benchmem -benchtime 5s -bench .`
 | ProtoHTTP1_1K  |  20000 |  383342 ns/op |  5.34 MB/s |  55044 B/op | 416 allocs/op |
 | ProtoHTTP1_64K |   3000 | 1932455 ns/op | 67.83 MB/s | 827150 B/op | 377 allocs/op |
 
+## go version devel +59129c6 Fri Sep 25 22:25:52 2015 +0000 darwin/amd64
```

Comment:
Add a comment that the difference comes from the use of hardware acceleration for AES-GCM.